draft-birkholz-scitt-architecture.md

```diff
@@ -121,13 +121,32 @@ Public SBOM ledger
   - can we keep track of the provenance of software artifacts?
   - can we share the cost of reviewing software across the industry?
 
+> While there is no need to limit SBOM to firmware, third-party firmware is a concrete example and may improve readability.
+
 ## Confidential Computing
 
-- Confidential Computing
-  - how can clients that connect to a CC service for the first time validate their attestation results?
-  - how to automatically patch a CC service? Trust, but audit later.
-  - automated software updates
-  - in particular for implementing SCITT services.
+Confidential Computing leverages trusted execution environments (TEEs)
+to operate services in isolation, with hardware-based integrity and
+confidentiality protection.  When clients connects to such a service,
+the service presents attestation evidence that typically includes a
+platform certificate for its TEE and a measurement of the software it
+runs in this TEE. The client verifies the evidence before using the
+service.
+
+But how can clients verify it is a valid measurement for the service?
```

Comment:
RATS defines the solution basically as "via a Verifier role". Evidence always comes with a proof of its Authenticity as it is signed by an Attesting Environment of the Attester (usually a RoT, that can be a layered Attesting Environment, sometimes called a Bootstrapped RoT).